packages/bff/src/model/validators.ts

```diff
@@ -1,15 +1,74 @@
 import { certifiedAttributesSatisfied } from "pagopa-interop-agreement-lifecycle";
-import { catalogApi, tenantApi } from "pagopa-interop-api-clients";
 import {
-  toDescriptorWithOnlyAttributes,
-  toTenantWithOnlyAttributes,
-} from "./api/apiConverter.js";
+  agreementApi,
+  catalogApi,
+  tenantApi,
+} from "pagopa-interop-api-clients";
+import { TenantId } from "pagopa-interop-models";
+import {
+  agreementApiState,
+  catalogApiDescriptorState,
+} from "./api/apiTypes.js";
+import { toDescriptorWithOnlyAttributes } from "./api/converters/catalogClientApiConverter.js";
+import { toTenantWithOnlyAttributes } from "./api/converters/tenantClientApiConverters.js";
+import { invalidEServiceRequester } from "./domain/errors.js";
+
+const SUBSCRIBED_AGREEMENT_STATES: agreementApi.AgreementState[] = [
```

Comment:
For consts like these we always used camelCase convention, see for example this file: https://github.com/pagopa/interop-be-monorepo/blob/13a3161d3fd58429ea3d8a93a49e9b72d777d39e/packages/agreement-process/src/model/domain/agreement-validators.ts#L57

```suggestion
const subscribedAgreementStates: agreementApi.AgreementState[] = [
```